static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -92,6 +92,11 @@ export const shouldIncludeOzone = (): boolean =>
     // include in 1 in 2 (50%) of page views
     !isInUsRegion() && !isInAuRegion() && getRandomIntInclusive(1, 2) === 1;
 
+export const shouldIncludeAppNexus = (): boolean =>
+    isInAuRegion() ||
+    ((config.get('switches.prebidAppnexusUkRow') && !isInUsRegion()) ||
+        !!pbTestNameMap().and);
```

Comment:
Do you need this `pbTestNameMap`? Looks complicated and there's already a check for `pbtest` in https://github.com/guardian/frontend/blob/884553b99f97ac36ac6463d17d8f78fda5d6a35e/static/src/javascripts/projects/commercial/modules/prebid/bid-config.js#L540

Reply:
Initially yes, because we want to test it in these regions with AppNexus UK by sending them a test URL. Once this gets released and OK'd then this can get removed.